sql-database/transparent-data-encryption/setup-tde-byok-sqlmi.ps1

```diff
@@ -16,11 +16,9 @@ New-AzResourceGroup -Name "MyRG" -Location $location
 # Create new Azure Key Vault with soft-delete option turned on (change name, RG and region): 
 New-AzKeyVault -VaultName "MyKeyVault" -ResourceGroupName "MyRG" -Location $location -EnableSoftDelete
 
-# Create Managed Service Identity for the Managed Instance: 
-$instance = Set-AzSqlManagedInstance -ResourceGroupName "MyRG" -Name "MyManagedInstance" -AssignIdentity
-
 # Authorize Managed Instance to use the AKV (wrap/unwrap key and get public part of key, if public part exists): 
```

Comment:
If the managed instance is required to have already been created, this should be made clear as a prerequisite at the top of the script.

Reply:
Added prereqs to the article: https://review.docs.microsoft.com/en-us/azure/sql-database/scripts/transparent-data-encryption-byok-sql-managed-instance-powershell?branch=pr-en-us-72002

Reply:
Please make this part of the script comments as well. Scripts are also used as external materials, not just on documentation.

Reply:
Done. Added comments to the top

![image](https://user-images.githubusercontent.com/40007119/55821215-d160bd00-5ab1-11e9-8740-0072557cf366.png)